src/app.jsx

```diff
@@ -509,6 +509,22 @@ const NavigatorCardBody = ({
                 });
             } else if (e.key === "Enter" && selected.length === 1) {
                 onDoubleClickNavigate(selected[0]);
+            } else if (e.key === "Delete") {
+                if (selected.length !== 0) {
+                    const currentDir = path.join("/") + "/";
+                    deleteItem(
+                        Dialogs,
+                        {
+                            selected,
+                            itemPath: currentDir + selected[0].name,
```

Comment:
We want to delete all selected items, not just the first one.

Reply:
https://github.com/cockpit-project/cockpit-navigator/blob/e6145bee4e56709b94d8770dba7593ed7bf6f126/src/apis/spawnHelpers.jsx#L37-L39
Yup that was pretty confusing to me. But why does it work correctly (removes multiple items without error)? And I checked how "the regular delete" is being done and It seems to pass only one item so I did the same.

Reply:
I don't understand most of the codebase So please correct me If I'm misunderstanding something

Reply:
@KKoukiou `itemPath` is ignored if multiple items are selected, so it's okay to pass `selected[0].name`. `spawnDeleteItem` handles multiple items.
The previous commit is works correctly, we can drop the latest commit. Let me know what you think!

`mutliple` is unused indeed, it's also present in the [main delete button,](https://github.com/cockpit-project/cockpit-navigator/blob/main/src/app.jsx#L215) we can remove it here or in a follow up.